cwl/src/test/scala/wdl4s/cwl/ParseBigThreeSpec.scala

```diff
@@ -2,81 +2,32 @@ package wdl4s.cwl
 
 
 import org.scalatest._
-import CwlCodecs._
+import CwlDecoder._
+import TestSetup._
 
 
 class ParseBigThreeSpec extends FlatSpec with Matchers {
   val namespace = "cwl"
 
   it should "parse 1st tool" in {
-  val firstTool = """
-cwlVersion: v1.0
-class: CommandLineTool
-baseCommand: echo
-inputs:
-- type: string
-  inputBinding:
-    position: 1
-  id: message
-outputs: []
-"""
 
-  decodeCwl(firstTool).isValid shouldBe true
+  decodeAllCwl(rootPath/"1st-tool.cwl").
+    value.
+    unsafeRunSync.
+    isRight shouldBe true
   }
 
   it should "parse first workflow" in {
-    val firstWorkflow = """
-cwlVersion: v1.0
-class: Workflow
-inputs:
-- type: string
-  id: ex
-- type: File
-  id: inp
-outputs:
-- type: File
-  outputSource: compile/classfile
-  id: classout
-steps:
-- run: cwl/src/test/resources/arguments.cwl
-  in:
-  - source: untar/example_out
-    id: compile/src
-  out:
-  - compile/classfile
-  id: compile
-- run: cwl/src/test/resources/tar-param.cwl
-  in:
-  - source: ex
-    id: extractfile
-  - source: inp
-    id: tarfile
-  out:
-  - example_out
-  id: untar
-"""
-    decodeCwl(firstWorkflow)
-      .isValid should be (true)
+    decodeAllCwl(rootPath/"1st-workflow.cwl").
+      value.
+      unsafeRunSync.
+      isRight shouldBe true
   }
 
   it should "parse env cwl" in {
-    val envCwl = """
-cwlVersion: v1.0
-class: CommandLineTool
-baseCommand: env
-requirements:
-- envDef:
-  - envValue: $(inputs.message)
-    envName: HELLO
-  class: EnvVarRequirement
-inputs:
-- type: string
-  id: file:///Users/danb/common-workflow-language/v1.0/examples/env.cwl#message
-outputs: []
-id: file:///Users/danb/common-workflow-language/v1.0/examples/env.cwl
-name: file:///Users/danb/common-workflow-language/v1.0/examples/env.cwl
-"""
-
-    decodeCwl(envCwl).isValid should be (true)
+    decodeAllCwl(rootPath/"env.cwl").
+      value.
+      unsafeRunSync.
+      isRight shouldBe true
```

Comment:
ToL (depending on how likely you think it is that this might break): boolean assertions can be tedious to debug. I find it better to do a `match` and nicely messaged:
```
case Left(_) => // great
case Right(error) => fail("Didn't do 'X' because $error")
```

Reply:
👍 I'll make these assertions more meaningful

Reply:
The message is provided in the description of the test, and thus writing out an error becomes essentially  `s"did not $description"`.  I think with a test consisting of a single assertion this layout is pretty reasonable as-is.